client/components/JestReact.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import App from './JestComponent';
+import '@testing-library/jest-dom';
+
```

Comment:
I would also group your imports following a consistent pattern with a line between the different types in this order 
1. builtin
2. external
3. internal
see https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m

Reply:
I think I organized it correctly. Not entirely sure what falls under builtin. Please check.